modin/pandas/test/test_series.py

```diff
@@ -4711,3 +4711,25 @@ def func(val):
     modin_series = modin_df["col"].apply(func)
 
     df_equals(modin_series, pandas_series)
+
+
+def test_apply_return_df():
+    modin_series, pandas_series = create_test_series([2, 4, 6, 0, 1])
+    eval_general(
+        modin_series,
+        pandas_series,
+        lambda series: series.apply(lambda x: pandas.Series([x, x + 1])),
+    )
+
+
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+@pytest.mark.parametrize("num_columns", [4, 8, 16, 32, 64])
+def test_apply_return_df_multiple_partitions(data, num_columns):
```

Comment:
```suggestion
def test_apply_return_df(data):
```

I don't see the value of testing multiple `num_columns`.